bundle/edu.gemini.itc.web/src/main/scala/edu/gemini/itc/web/ITCRequest.scala

```diff
@@ -60,6 +60,15 @@ sealed abstract class ITCRequest {
       }
   }
 
+  /** Gets the named value as an integer. */
+  def intParameterCoadd(name: String): Int = parameter(name).trim() match {
+    case ""   => 0
+    case i    =>
+      try i.toInt catch {
+        case _: IllegalArgumentException => 1
```

Comment:
so if there is an error we default to 1. Is that the desired behavior?

Reply:
Oops, this method is actually not used anywhere, I'll remove it.